imports/plugins/included/product-admin/client/components/ProductTable.js

```diff
@@ -105,24 +87,14 @@ function ProductTable({ onCreateProduct }) {
     disableClick: true
   });
 
-  const classes = useStyles();
-
-  let displayCard;
-  let displayButton;
-  if (isClosed === true) {
-    displayCard = "none";
-    if (isFiltered === true) {
-      displayButton = "none";
-    } else {
-      displayButton = "block";
-    }
-  } else {
-    displayCard = "block";
-    displayButton = "none";
-  }
+  const handleShowFilterByFile = (isVisible) => {
```

Comment:
prefer regular `function` definitions rather than arrow functions in functional components

Reply:
I know that we discussed preferring functions over arrow functions, however, now I am leaning towards using arrows functions in certain cases, such as the function above. It's a two-line function that only toggles a state value. If  I refactor it to a regular function, then our lint rules require documentation which would add to the size of this file unnecessarily, and it would not add much value to the function as it's so small, that its documentation is superfluous.

Reply:
You're right, I've talked with others about it and even seen for myself the jsdoc complaints on functions in functions needing comments. I think we need to reassess our strategy on function declarations inside functions for these React components to prefer arrow functions.

I looked through the eslint rules on this and couldn't find a way to make an exception to rules for this situation.

For now, leave it how it is, we'll have to discuss what we do going forward.

Reply:
OK, I am leaning towards using arrow functions inside functional components.